### PR TITLE
Finished lesson09

### PR DESCRIPTION
**Pull request by githubtater**

### CODE REVIEW

**Comment by githubtater** [comment]
I modified and moved my .gitignore file. 
Should be good to go now.

**Comment by natasha-aleksandrova** [comment]
@githubtater it looks like you removed the whole file which created a merge conflict.

File: students/Roy_Tate/lesson08/circle.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+class Circle:
+
+    def __init__(self, radius):
+        '''Create a circle with a radius. Radius is mandatory (cant have a circle without one)'''
+        self._radius = radius
+
+    @property
+    def radius(self):
+        '''Get radius of circle'''
```
Comment (natasha-aleksandrova) [inline students/Roy_Tate/lesson08/circle.py]: note that python docstrings use double triple quotes `"""Example"""`

File: students/Roy_Tate/lesson09/mailroom_ui.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        print('\n{:*^20}'.format(' MAIN MENU '))
+        while True:
+            for k, v in sorted(main_menu_args.items()):
+                print(k, v['option'])
+            response = input('\nEnter your selection: \n').strip()
+            self.menu_selection(main_menu_args, response)
+
+
+
+    def die(self):
```
Comment (natasha-aleksandrova) [inline students/Roy_Tate/lesson09/mailroom_ui.py]: note that class methods only need one line space between methods

**Pull request closed without merging**